report_pre_thesis/src/sections/experiments/pls_outlier.tex

```diff
@@ -0,0 +1,32 @@
+\subsection{Experiment: PLS Automated Outlier Removal}\label{sec:experiment_pls_automated_outlier_removal}
+The original PLS1-SM identified outliers manually by inspecting the leverage and spectral residuals plots.
+We have instead chosen to automate this based on the reasons described in Section~\ref{sec:methodology_outlier_removal}.
+Examining the performance implications of completely omitting outlier removal would be worthwhile.
```

Comment:
In this format the sentence feels disconnected to the previous/following sentence.
Perhaps this is better?
```suggestion
As such, examining the performance implications of completely omitting outlier removal would be worthwhile.
```